src/DynamoCoreWpf/Views/Core/NodeView.xaml

```diff
@@ -78,9 +78,14 @@
                               CommandParameter="CrossProduct"
                               Header="{x:Static p:Resources.ContextMenuLacingCrossProduct}" />
                 </MenuItem>
+                <MenuItem Name="nodeIsFrozen"
+                          Header="{x:Static p:Resources.NodesRunStatus}"
+                          IsChecked="{Binding NodeRunChecked,Mode=OneWay}"   
+                          IsEnabled="{Binding NodeRunEnabled, Mode=OneWay}"
+                          Command="{Binding Path=ComputeRunStateOfTheNodeCommand}" />
```

Comment:
This is the FREEZE option on the nodeview. The option is checkable  according to the requirement. It is not an action. I added FREEZE option to Workspace Context menu and Dynamo Menubar. this is  a placeholder when we need to select bunch of nodes and made to freeze.